ext/flexmailer/tests/phpunit/Civi/FlexMailer/ClickTracker/TextClickTrackerTest.php

```diff
@@ -45,19 +45,14 @@ public function getHrefExamples() {
       '<p><a href=\'tracking(https://sub.example.com/foo.php?whiz=%2Fbang%2F&pie[fruit]=apple)\' rel=\'nofollow\'>Foo</a></p>',
     ];
     $exs[] = [
-      // Messy looking URL, designed to trip-up quote handling
```

Comment:
Because we previously converted links into [Text] plus a footnote with the URL, all we were testing here was of `[Foo] === [Foo]` over and over. Now that the URL is inline in parentheses, these tests actually do something (and so they can fail).

The two javascript:alert ones shouldn't have tracking prepended, because the regex we're using only matches urls with http in them, so these don't count as urls and don't get the tracking. Probably not so valuable, but we can leave them in here just to make sure the tracking isn't added.

I've removed the one with a newline in the url. I don't think that's a reasonable thing to test or there is any need for us to support a url with a newline in it (url-encoded, sure, but a literal `/n`, no).